### DISCUSSION TITLE
Default action for filename conflicts

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3635] [Feature Requests]
### Suggest an idea for ExpressionEngine

Right now editors have to choose what they want to do with the image name: append a numeral, rename, replace the older file, or cancel uploading.

And guess what :) All of my editors choose "Append numeral" each time.

Maybe we can make their work a little easier by having a default action for those filename conflicts? And, maybe, for those, who do not care about filenames, we can add one more option - renaming all the files using a template, like `{date}/{url_title}-{hash}.{ext}`?

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/3699
